.github/workflows/build_wheels.yml

```diff
@@ -130,12 +130,20 @@ jobs:
 
       - uses: actions/upload-artifact@v4
         with:
-          name: macos-${{ matrix.cibw_arch }}-wheels
+          name: macosx_${{ matrix.cibw_arch }}-wheels
           path: ./wheelhouse/*.whl
 
+  build_and_test_wheels:
```

Comment:
Maybe we could add a comment explaining why it is useful to have that job there, since it literally does nothing except making it easier to set required checks.

Perhaps we could call it something different? Maybe something like `wait_for_required_jobs`.

Reply:
I added the comment, but as we discussed offline I left the name the same because it would be strange to have:
```yaml
  upload_pypi:
    needs: [wait_for_required_jobs, build_sdist]
```
instead of
```yaml
  upload_pypi:
    needs: [build_and_test_wheels, build_sdist]
```